pages/common/sponge.md

```diff
@@ -1,7 +1,7 @@
 # sponge
 
 > Soak up the input before writing the output file.
-> More information: <https://linux.die.net/man/1/sponge>.
+> More information: <https://manpages.debian.org/moreutils/sponge.1.html>.
```

Comment:
maybe we can use latest, to redirect to the latest man page version automatically:
```suggestion
> More information: <https://manpages.debian.org/latest/moreutils/sponge.1.en.html>.
```

Reply:
Oh forget it. It's the same behavior with your URL.

Reply:
I don't think we need this, because mitting `latest/` redirects to the latest version too.

Reply:
On that note, the debian manpage link in the other PRs need to be updated to this too, because they have the version right in the link, like `buster`, `stretch`, etc.